Kanban/Services/KBNProjectService.m

```diff
@@ -236,4 +241,47 @@ - (void)getProjectsUpdate {
 
 }
 
+-(void)syncProjectsOnParse {
+    [[KBNCoreDataManager sharedInstance] getProjectsOnSuccess:^(NSArray *records) {
```

Comment:
improvement: maybe get projects from the last time that was connected not all the projects